e2e/atls/snp/snp_test.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2025 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+//go:build e2e
+
+package snp
```

Comment:
imo this shouldn't be a SNP specific test package. Instead you could just skip the snp test on platforms that don't support snp and make it a generic atls test.

Reply:
Sure, I also wondered about how we might want to extend this to TDX in the future. I originally thought about 2 packages, but your suggestion makes sense.